server/src/main/java/com/ti/server/signup/service/SignUpService.java

```diff
@@ -9,15 +9,15 @@
 
 @Service
 @RequiredArgsConstructor
-public class SignUpService
-{
-    private final UserRepository userRepository;
 
+public class SignUpService {
+    private final UserRepository userRepository;
     public ResponseEntity<Object> saveUserInfo(UserEntity userEntity) {
```

Comment:
The space I put above this line was intentional....It's a little hard to see that there is a new method if the defining of the userRepository object is directly above it.